### ISSUE TITLE
assert in `bombing-fools.cfg`

### ISSUE THREAD

**Issue by rkettering** [frogatto/frogatto#529]
```2022-02-14 05:57:53.417 Frogatto[1006:142295] CRITICAL: formula.cpp:233 ASSERTION FAILED: custom_object.cpp:3810 ASSERTION FAILED: ILLEGAL OBJECT ACCESS WITH STRICT CHECKING IN moth_bomber_bomb: collide_with At   FRAME 0: if(in_change_dimensions_fail > 15 or (not collide_with), die_from_bad_collision_state,
	          [add(in_change_dimensions_fail, 1),
			   add(velocity_x,if(collide_with.midpoint_x > midpoint_x, -500,500))
			  ]) where collide_with = obj hittable <- arg.collide_with


ERROR ENCOUNTERED WHILE RUNNING COMMAND GENERATED BY THIS EXPRESSION:
At modules/frogatto4/data/object_prototypes/enemies_specialized/moth_bomber.cfg 50:
set(the_bomb.collides_with_level, true),
^--------------------------------------^

2022-02-14 05:57:53.417 Frogatto[1006:142295] CRITICAL: 
---
2022-02-14 05:57:53.417 Frogatto[1006:142295] CRITICAL: stack trace:
2022-02-14 05:57:53.453 Frogatto[1006:142295] CRITICAL:   1   Frogatto                            0x00000001000db0b5 _Z16output_backtracev + 53
2022-02-14 05:57:53.454 Frogatto[1006:142295] CRITICAL:   2   Frogatto                            0x0000000100624950 _ZNK10game_logic15CommandCallable10runCommandERNS_15FormulaCallableE + 464
2022-02-14 05:57:53.454 Frogatto[1006:142295] CRITICAL:   3   Frogatto                            0x0000000100355170 _ZN12CustomObject14executeCommandERK7variant + 160
2022-02-14 05:57:53.454 Frogatto[1006:142295] CRITICAL:   4   Frogatto                            0x0000000100355147 _ZN12CustomObject14executeCommandERK7variant + 119
2022-02-14 05:57:53.454 Frogatto[1006:142295] CRITICAL:   5   Frogatto                            0x0000000100355147 _ZN12CustomObject14executeCommandERK7variant + 119
2022-02-14 05:57:53.454 Frogatto[1006:142295] CRITICAL:   6   Frogatto                            0x0000000100354948 _ZN12CustomObject19handleEventInternalEiPKN10game_logic15FormulaCallableEb + 824
2022-02-14 05:57:53.454 Frogatto[1006:142295] CRITICAL:   7   Frogatto                            0x0000000100354d7e _ZN12CustomObject11handleEventEiPKN10game_logic15FormulaCallableE + 286
2022-02-14 05:57:53.454 Frogatto[1006:142295] CRITICAL:   8   Frogatto                            0x000000010033f7c2 _ZN12CustomObject13staticProcessER5Level + 178
2022-02-14 05:57:53.454 Frogatto[1006:142295] CRITICAL:   9   Frogatto                            0x000000010033cbca _ZN12CustomObject7processER5Level + 12970
2022-02-14 05:57:53.454 Frogatto[1006:142295] CRITICAL:   10  Frogatto                            0x0000000100a8c757 _ZN5Level13do_processingEv + 519
2022-02-14 05:57:53.454 Frogatto[1006:142295] CRITICAL:   11  Frogatto                            0x0000000100a8c23f _ZN5Level7processEv + 383
2022-02-14 05:57:53.454 Frogatto[1006:142295] CRITICAL:   12  Frogatto                            0x0000000100b6f4d6 _ZN11LevelRunner10play_cycleEv + 15542
2022-02-14 05:57:53.454 Frogatto[1006:142295] CRITICAL:   13  Frogatto                            0x0000000100b6b121 _ZN11LevelRunner10play_levelEv + 657
2022-02-14 05:57:53.454 Frogatto[1006:142295] CRITICAL:   14  Frogatto                            0x0000000100bbc247 main + 17847
2022-02-14 05:57:53.454 Frogatto[1006:142295] CRITICAL:   15  libdyld.dylib                       0x00007fff690823d5 start + 1
2022-02-14 05:57:53.454 Frogatto[1006:142295] CRITICAL: ---
2022-02-14 05:57:53.454 Frogatto[1006:142295] INFO: asserts.cpp:85 : ATTEMPTING TO SEND CRASH REPORT...```

**Comment by rkettering** [comment]
Possibly also:  
<img width="472" alt="image" src="https://user-images.githubusercontent.com/863312/157594253-10ccc5cd-3323-4c64-a808-f721cf29ca74.png">

**Comment by rkettering** [comment]
Possibly fixed in:  1f204455a

**Comment by rkettering** [comment]
Fixed in 87afa7201
